### PR TITLE
Add documentation for Analyses and Timeframes to README.md.

### LINKED ISSUE #18: Add example documentation for Queries
It looks like there's code for query support, but there are no examples of how to run them!

### PR DESCRIPTION
**Pull request by baumatron**
This adds some documentation for Analysis and Timeframes.

Fixes #18.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,6 +190,24 @@ The server may reject one or more events included in the cache. If this happens
...
+To run [analyses](https://keen.io/docs/api/#analyses) on your data, use the provided `KeenClient.Query` family of methods. For example:
+
+```
+var itemCount = keenClient.Query(QueryType.Count(), "target_collection", null);
+```
+
+An `async` version of this analysis could be run as follows:
+
+```
+var itemCount = await keenClient.Query(QueryType.Count(), "target_collection", null);
```
Comment (masojus) [inline README.md]: QueryAsync()?

File: README.md [hunk 2]
```diff
@@ -190,6 +190,24 @@ The server may reject one or more events included in the cache. If this happens
...
+```
+
+Additional qualifiers can be added to the analysis, such as the target property to use for analyses that require it. A timeframe or list of filters to use for the analysis can also be provided. If you'd like to get results in a grouped or time interval format, the `KeenClient.QueryGroup`, `KeenClient.QueryInterval`, and `KeenClient.QueryIntervalGroup` synchronous and asynchronous methods can be used. See the Grouped and Interval Query Results and Filters sections below for more detail.
```
Comment (masojus) [inline README.md]: "A timeframe and/or list of filters" right? They aren't mutually exclusive, so the "or" alone seems a little strange.

File: README.md [hunk 3]
```diff
@@ -240,6 +258,27 @@ var registeredUsers = result.ElementAt(0);
 var registeredAndSubscribedUserCount = result.ElementAt(1);
 ```
 
+Timeframes
+------------
+
+A timeframe can be specified for analysis using the `QueryRelativeTimeframe` and `QueryAbsoluteTimeframe` classes, along with an optional `timezone` parameter passed to the `KeenClient.Query` method when using `QueryRelativeTimeframe`. The `timezone` parameter must be one of the timezones supported by the Keen.io API as specified [here](https://keen.io/docs/api/#timezone).
```
Comment (masojus) [inline README.md]: Maybe "Keen.IO" to have capitalization consistent with elsewhere in the docs.

File: README.md (hunk 2 above)
Comment (baumatron) [inline README.md]: Right, I'll make it more precise.

File: README.md (hunk 3 above)
Comment (baumatron) [inline README.md]: I'll actually change this to Keen IO, since that seems to be how it's documented elsewhere.

**Pull request merged**